### DISCUSSION TITLE
B.C. dates

### DISCUSSION THREAD

**Discussion by pajaro1966** [mermaid-js/mermaid#4643] [Q&A]
Dear friends,

I want to use Gantt graphs for representing historical facts instead of Timeline graphs, because Gantt graphs offers more possibilities to represent different timelines simultaneously. However, I have difficulties to represent dates Before the Common Era (BCE) and to mix them with Common Era (CE) dates. I thought that just adding a minus ("-") sign before the dates would work, but it doesn't. I read the documentation, but couldn't find this topic. Could you help me out? Is there any way?
Thanks in advance for your help.
A.R.

**Comment by renatalrt** [comment]
It would be really helpful! I agree!

It would be so helpful if the BCE dates would work on Mermaid in most browsers.
I am doing an Art History Timeline and I just found out that it only works on Safari. I think it would be amazing for other students, teachers and historians to have this possibility.

Here is a snippet of the code I used. It works perfectly on Safari, but not on Chrome.

```
gantt
    title History Timeline
    dateFormat  YYYY

    section Ancient Egypt
    King Menes Rule    :milestone, -3200, 0d
    King Cheops     :milestone, -2500, 0d
    Construction of Great Pyramid of Cheops     :done, -2500, 30y
    King Akhenaton     : -1351, -1334
```
